### DISCUSSION TITLE
0001 - MEP Purpose and Guidelines

### DISCUSSION THREAD

**Discussion by chrisjsewell** [jupyter-book/myst-enhancement-proposals#2] [MEP]
Feedback for MEP 0001.

HackMD document for draft is here: https://hackmd.io/8JnI4N82QKO84oo4rT6YMg

**Comment by rowanc1** [comment]
As discussed in the recent [team meeting](https://github.com/executablebooks/meta/issues/831), we will be editing and testing out the process before settling on anything!

Doc: https://hackmd.io/8JnI4N82QKO84oo4rT6YMg
